include/sta/Network.hh

```diff
@@ -313,6 +313,9 @@ public:
 		const Instance *hier_inst) const;
   bool isDriver(const Pin *pin) const;
   bool isLoad(const Pin *pin) const;
+  bool isClock(const Pin *pin) const;
+  bool isRiseEdgeTriggered(const Pin *pin) const;
+  bool isFallEdgeTriggered(const Pin *pin) const;
```

Comment:
I don't think these belong in the network api.
They are not very reliable given the various possible sequential clocked_on expression and certainly don't work with statetables (good lock).